airflow/operators/python.py

```diff
@@ -178,6 +178,19 @@ class BranchPythonOperator(PythonOperator, SkipMixin):
 
     def execute(self, context: Dict):
         branch = super().execute(context)
+        # TODO: The logic should be moved to SkipMixin to be available to all branch operators.
+        if isinstance(branch, str):
```

Comment:
Should we also change SkipMixin while we are changing it  ? (see the comment above)

Reply:
Seperate PR.
I will work on it. We want also to change the functionality so it will accept None and skip all downstream tasks (untill a junction/join task).